test/e2e/test/kibana/builder.go

```diff
@@ -176,6 +176,22 @@ func (b Builder) WithTLSDisabled(disabled bool) Builder {
 	return b
 }
 
+func (b Builder) WithAPMIntegration() Builder {
+	if version.MustParse(b.Kibana.Spec.Version).LT(version.MinFor(8, 0, 0)) {
+		// configuring APM integration is not necessary below 8.0.0, no-op
+		return b
+	}
+
+	return b.WithConfig(map[string]interface{}{
```

Comment:
Maybe I'm missing something obvious but how does it work for tests which do not imply Kibana ? (for example `TestCrossNSAssociation`)